platform-sdk/docs/proposals/001-tss-rosters-design-proposal/proposal.md

```diff
@@ -144,7 +151,8 @@ This proposal introduces four types of `Roster` - `active`, `candidate`, `genesi
 2. A `Genesis Roster` is loaded from disk at the genesis of a new network and becomes the first active roster.
 3. The `Candidate Roster` is a roster that will become active at the next software upgrade.
 4. An `Override Roster` is loaded from disk and unconditionally set as the latest active roster. This is used to
-   transplant the state to a different network.
+   transplant the state to a different network.  Until the TSS effort is delivered and the network is given a ledger
+   id, this override mechanism can be used to manually correct the roster / address book on the current network.
```

Comment:
Minor: I'd suggest to note that using the override roster would/should still be possible as an emergency mechanism, even after the TSS is enabled, with an understanding that the ledger id cannot be preserved in that case.

Reply:
I think that is the domain of a later TSS Proposal and needs to be thought through more thoroughly than what we can say here.